client/tests/integration/manuscript-manager-template-test.coffee

```diff
@@ -23,7 +23,10 @@ module 'Integration: Manuscript Manager Templates',
   setup: ->
     app = startApp()
     server = setupMockServer()
-
+    Tahi.__container__.lookup(
+      'controller:admin/journal/manuscript-manager-template/edit'
+    )._actions.saveTemplateOnClick = -> console.log 'No Action'
+      
```

Comment:
Hm, why is this necessary? Is the original function restored after the test finishes?